.circleci/config.yml

```diff
@@ -217,7 +212,6 @@ jobs:
       - run: ./scripts/publish-npm-semver-tagged
 
 workflows:
-  version: 2
```

Comment:
the CircleCI VSCode plugin told me that this key was deprecated / no longer necessary